packages/appstore/src/components/containers/__tests__/grid-container.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import GridContainer from '../grid-container';
+import { render, screen } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+describe('GridContainer', () => {
+    it('should render children', () => {
+        const store = mockStore({});
+        render(
+            <StoreProvider store={store}>
+                <GridContainer>
+                    <div>
+                        <h1>Traders hub</h1>
```

Comment:
As we are using the same text for assertion and `h1`, so can we make a const variable and use it in both places.
```suggestion
                        <h1>{traders_hub_label}
                        </h1>
```